post_processing/utils/archive_handler.py

```diff
@@ -0,0 +1,372 @@
+"""
+Archive Handler Utility
+
+Handles extraction of Zathras result archives:
+- results_{test}.zip → results_{test}_.tar → test result files
```

Comment:
@dvalinrh did we go back to a regular tarball instead of a zip -> tar -> test results?